code/__DEFINES/misc.dm

```diff
@@ -141,7 +141,7 @@
 //2 = reduced hud (just hands and intent switcher)
 //3 = no hud (for screenshots)
 
-#define MINERAL_MATERIAL_AMOUNT 2000
+#define MINERAL_MATERIAL_AMOUNT 1000
```

Comment:
Well who the fuck thought 2k was a good idea? Where does that number even come from?